frontend/src/routes/users/me/+page.svelte

```diff
@@ -0,0 +1,252 @@
+<script lang="ts">
+  import user from '$stores/user';
+  import wretch from 'wretch';
+
+  const api = wretch('http://localhost:3000/api');
+
+  let birthdate: string;
+  let isSuccess = false;
+  $: console.log(birthdate);
+  $: console.log(isSuccess);
+  $: isSuccess;
+
+  const submitBirthdate = async () => {
+    const response = await api
+      .url('/v1/users/me')
+      .headers({ Authorization: `Bearer ${$user.token}` })
+      .post({ birthdate });
+
+    if (response) {
+      isSuccess = true;
+    }
+  };
+
+  const closeAlert = () => {
+    isSuccess = false;
+  };
+</script>
+
+<div class="lg:grid lg:grid-cols-12 lg:gap-x-5 container mx-auto mt-2">
+  <aside class="py-6 px-2 sm:px-6 lg:col-span-3 lg:py-0 lg:px-0">
+    <nav class="space-y-1">
+      <!-- Current: "bg-gray-50 text-indigo-700 hover:text-indigo-700 hover:bg-white", Default: "text-gray-900 hover:text-gray-900 hover:bg-gray-50" -->
+      <a
+        href="#"
+        class="bg-gray-50 text-indigo-700 hover:text-indigo-700 hover:bg-white group rounded-md px-3 py-2 flex items-center text-sm font-medium"
+        aria-current="page"
+      >
+        <!--
+          Heroicon name: outline/user-circle
+
+          Current: "text-indigo-500 group-hover:text-indigo-500", Default: "text-gray-400 group-hover:text-gray-500"
+        -->
+        <svg
+          class="text-indigo-500 group-hover:text-indigo-500 flex-shrink-0 -ml-1 mr-3 h-6 w-6"
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor"
+          aria-hidden="true"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z"
+          />
+        </svg>
```

Comment:
Let's move these icons to the separate components.
Examples: `frontend/src/components/icons/Bars3/Outline.svelte` and use it like `<Icon name="Bars3">`